packages/pinia/__tests__/lifespan.spec.ts

```diff
@@ -168,4 +164,26 @@ describe('Store Lifespan', () => {
 
     destroy()
   })
+
+  it('dispose stops store reactivity', () => {
+    const n = ref(0)
+    const pinia = createPinia()
+    setActivePinia(pinia)
+    const inStoreWatch = vi.fn()
+
+    const useStore = defineStore('a', () => {
+      watch(n, inStoreWatch, {
+        flush: 'sync',
+      })
+      return { n }
+    })
```

Comment:
```suggestion
    const pinia = createPinia()
    setActivePinia(pinia)
    const inStoreWatch = vi.fn()

    const useStore = defineStore('a', () => {
      const n = ref(0)
      watch(n, inStoreWatch, {
        flush: 'sync',
      })
      return { n }
    })
```

we don't need to put the n outside in this case, it will only create a small leak in the tests